enmanuel-delanuez/short-response.md

```diff
@@ -0,0 +1,71 @@
+# Problem Set 2.2 - Variables and Control Flow
+## Short Response Questions
+
+1. **What does the code below log? Why?**
+   ```javascript
+      let singer = 'Omar Apollo';
+      console.log(`My favorite singer is ${singer}`);
+   ```
+   This logs `'My favorite singer is Omar Apollo'` because the use of `${}` interpolates the value of a binding into the string.
+
+2. **What happens when we run the following code? Why?**
+   ```javascript
+   const favorite = 'Juan Pablo';
+   console.log(`Our favorite Marcy Lab family member is ${favorite}!`);
+
+   favorite = 'Maya';
+   console.log(`Actually, ${favorite} is my favorite.`);
+   ```
+   This logs `'Our favorite Marcy Lab family member is Juan Pablo'` because the binding `favorite` was declared with the keyword `const`, meaning it is a constant that can't be reassigned.
```

Comment:
Yes it logs this and...
Run this in the console and see what happens _after_ the first statement is logged. Report back with what and why.